documentation-site/components/yard/config/progress-bar.ts

```diff
@@ -53,6 +53,12 @@ const ProgressBarConfig: TConfig = {
         },
       },
     },
+    stepped: {
```

Comment:
to me, `stepped` implies that it's a Boolean value. Should we name it `steps`?

Reply:
I don't have a strong opinion on the name. 'stepped' is the term design uses if we want to stay aligned. Also, I'd like to avoid introducing two props for it like one that defines it to be stepped and the second defines the number of steps. Otherwise, I'm ok with renaming it.

Reply:
Yeah, I'd just rename it, and wouldn't introduce another prop